models/classes/cat/CatUtils.php

```diff
@@ -94,8 +94,8 @@ public static function isAssessmentSectionAdaptive(AssessmentSection $section, $
         }
         
         $isAdaptive = false;
-        
-        if (($selection = $section->getSelection()) !== null && (($xmlExtension = $selection->getXml())) !== null) {
+        $xmlExtension = new DOMDocument();
```

Comment:
could you please apply or import it
```suggestion
        $xmlExtension = new \DOMDocument();
```